ping_test.go

```diff
@@ -1,4 +1,4 @@
-package ping_test
```

Comment:
I do prefer having a separate package for testing libraries unless i'm testing unexported functions directly, which is something that could be done..

Reply:
Yes, you are right! I've changed it to the origin.